Signal/src/ViewControllers/AddToGroupViewController.m

```diff
@@ -0,0 +1,127 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "AddToGroupViewController.h"
+#import "BlockListUIUtils.h"
+#import "ContactsViewHelper.h"
+#import "OWSContactsManager.h"
+#import "SignalAccount.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AddToGroupViewController () <SelectRecipientViewControllerDelegate>
+
+@end
+
+#pragma mark -
+
+@implementation AddToGroupViewController
+
+- (void)loadView
+{
+    self.delegate = self;
+
+    [super loadView];
+
+    self.title = NSLocalizedString(@"ADD_GROUP_MEMBER_VIEW_TITLE", @"Title for the 'add group member' view.");
+}
+
+- (NSString *)phoneNumberSectionTitle
+{
+    return NSLocalizedString(@"ADD_GROUP_MEMBER_VIEW_PHONE_NUMBER_TITLE",
+        @"Title for the 'add by phone number' section of the 'add group member' view.");
+}
+
+- (NSString *)phoneNumberButtonText
+{
+    return NSLocalizedString(@"ADD_GROUP_MEMBER_VIEW_BUTTON",
+        @"A label for the 'add by phone number' button in the 'add group member' view");
+}
+
+- (NSString *)contactsSectionTitle
+{
+    return NSLocalizedString(
+        @"ADD_GROUP_MEMBER_VIEW_CONTACT_TITLE", @"Title for the 'add contact' section of the 'add group member' view.");
+}
+
+- (void)phoneNumberWasSelected:(NSString *)phoneNumber
+{
+    OWSAssert(phoneNumber.length > 0);
+
+    __weak AddToGroupViewController *weakSelf = self;
+    ContactsViewHelper *helper = self.contactsViewHelper;
+    if ([helper isRecipientIdBlocked:phoneNumber]) {
+        [BlockListUIUtils showUnblockPhoneNumberActionSheet:phoneNumber
+                                         fromViewController:self
+                                            blockingManager:helper.blockingManager
+                                            contactsManager:helper.contactsManager
+                                            completionBlock:^(BOOL isBlocked) {
+                                                if (isBlocked) {
+                                                    [weakSelf addToGroup:phoneNumber];
+                                                }
+                                            }];
+    } else {
```

Comment:
> be unselectable

Done. 

> show an alert that says "You cannot remove group members

Well, this dialog is used for adding members to groups, not removing, but you're right, they shouldn't be selectable. Done.  I am however adding the warning you're suggesting to the "edit group" view.

I've applied similar changes to the "add to block list" view. thanks.